src/util-inl.h

```diff
@@ -294,6 +294,11 @@ std::string ToLower(const std::string& in) {
   return out;
 }
 
+inline void ToLowerInPlace(std::string& in) {
```

Comment:
C++ style guides for Node.js states that you should always pass the pointer to a string, a reference is not allowed.

Reply:
will remove the actual utility as ended up not using it